### ISSUE TITLE
feat(ACCESSIBILITE): 10.9 : Dans chaque page web, l'information ne doit pas être donnée uniquement par la forme, taille ou position. Cette règle est-elle respectée ?

### ISSUE THREAD

**Issue by celinelr** [SocialGouv/domifa#2614]
## P07 : https://domifa.fabrique.social.gouv.fr/nouveau (tout le processus de création)

Au moins une information donnée uniquement par la forme, la taille ou la position n'a pas d'alternative.
Par exemple :
- L'indication d'étape active et étape terminée

## Correction :
- Implémenter un texte caché hors écran précédant le nom de l'étape en suivant ce modèle :
Pour l'étape en cours : <span>1</span> <span class="sr-only">En cours : </span> <span>État civil</span>
Pour les étapes terminées : <span>1</span> <span class="sr-only">Terminé : </span> <span>État civil</span>

**Comment by pYassine** [comment]
@celinelr il me semble que c'est ok

**Comment by celinelr** [comment]
> @celinelr il me semble que c'est ok

Non, on me propose d'aller à l'étape 1 mais je ne sais pas où je me situe. Lorsque je suis à l'étape 2 en passant le focus sur l'étape 1 on me propose d'aller à l'étape 1 sans m'indiquer que l'étape 1 est terminée et lorsque je que clique de nouveau sur "Tab" on me propose d'aller à l'étape 2 sans m'informer que j'y suis déjà.

Je te fais la demo au sprint planning

**Comment by Chirazh** [comment]
Quand je mets le guide vocal et que je teste sur un dossier en cours de renouvellement ça m'indique pour chaque étape du formulaire où je suis mais je n'ai pas le détail de ce qui vient avant ou après comme tu me l'as expliqué à l'oral

**Comment by Chirazh** [comment]
Création de demande, étape 1 sur 5, état-civil
